server/api.go

```diff
@@ -385,6 +378,39 @@ func (a *API) getLinkedChannels(w http.ResponseWriter, r *http.Request) {
 	a.writeJSONArray(w, http.StatusOK, paginatedLinks)
 }
 
+func (a *API) getMSTeamsTeamList(w http.ResponseWriter, r *http.Request) {
+	userID := r.Header.Get(HeaderMattermostUserID)
+	teams, statusCode, err := a.p.GetMSTeamsTeamList(userID)
+	if err != nil {
+		http.Error(w, "Error occurred while fetching the MS Teams team list.", statusCode)
```

Comment:
If the error is because the user is not connected, shouldn't we show that error here?